### PR TITLE
Bugfix #80

### PR DESCRIPTION
**Pull request by nischalsamji**
. Added bug-fix for handling multiple date formats.

### CODE REVIEW

**Comment by erickpeirson** [comment]
@nischalsamji This looks like a great start! Could you go ahead and write some tests to show that your fix behaves as expected?

File: tethne/readers/zotero.py [hunk 1]
```diff
@@ -80,11 +80,13 @@ def handle_link(self, value):
                 return str(o).replace('file://', '')                 
 
     def handle_date(self, value):
-        try:
-            return iso8601.parse_date(str(value)).year    
-        except iso8601.ParseError:
-            return datetime.strptime(str(value), "%m/%d/%Y").date().year
-        
+    		for datefmt in ("%B %d, %Y", "%Y-%m", "%Y-%m-%d", "%m/%d/%Y"):
```
Comment (erickpeirson) [inline tethne/readers/zotero.py:83]: Is this indentation correct? Looks like it's several levels too deep

File: tethne/readers/zotero.py [hunk 2]
```diff
@@ -80,11 +80,13 @@ def handle_link(self, value):
...
+		        try:
+		            print datetime.strptime(str(value), datefmt).date().year
+		        except iso8601.ParseError:
```
Comment (erickpeirson) [inline tethne/readers/zotero.py:86]: How would an iso8601.ParseError occur if you don't use any methods from the iso8601 module?

**Comment by nischalsamji** [comment]
Think I missed a line. Fixing it.

File: tethne/readers/zotero.py (hunk 1 above)
Comment (nischalsamji) [inline tethne/readers/zotero.py:83]: The Indentation looks fine for me in my editor. Rechecking.

**Pull request closed without merging**